engine/runtime/src/main/java/org/enso/interpreter/runtime/error/ConstructorDoesNotExistException.java

```diff
@@ -0,0 +1,31 @@
+package org.enso.interpreter.runtime.error;
```

Comment:
This exception only makes sense in the context of the server, so please move it there.